gem/templates/chhaajja/404.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block body_class %}template-404{% endblock %}
+
+{% block content %}
+    <h1>Page not found</h1>
+
+    <h2>Sorry, this page could not be found.</h2>
```

Comment:
You should wrap static text with {% trans "static text here" %}  on the templates in order to generate a .po file Translations.